### PR TITLE
Move lock timestamp pref to AppPreferences

### PR DESCRIPTION
**Pull request by ezaquarii**
Signed-off-by: Chris Narkiewicz <hello@ezaquarii.com>

### CODE REVIEW

File: src/main/java/com/owncloud/android/authentication/PassCodeManager.java [hunk 1]
```diff
@@ -78,7 +79,8 @@ public void onActivityCreated(Activity activity) {
     }
 
     public void onActivityStarted(Activity activity) {
-        Long timestamp = PreferenceManager.getLockTimestamp(activity);
+        AppPreferences preferences = PreferenceManager.fromContext(activity);
+        Long timestamp = preferences.getLockTimestamp();
```
Comment (AndyScherzinger) [inline src/main/java/com/owncloud/android/authentication/PassCodeManager.java]: could be shortened to `PreferenceManager.fromContext(activity).getLockTimestamp()`

File: src/main/java/com/owncloud/android/authentication/PassCodeManager.java [hunk 2]
```diff
@@ -110,7 +112,8 @@ public void onActivityStopped(Activity activity) {
     }
 
     private void setUnlockTimestamp(Activity activity) {
-        PreferenceManager.setLockTimestamp(activity, System.currentTimeMillis());
+        AppPreferences preferences = PreferenceManager.fromContext(activity);
+        preferences.setLockTimestamp(System.currentTimeMillis());
```
Comment (AndyScherzinger) [inline src/main/java/com/owncloud/android/authentication/PassCodeManager.java]: could be shortened to `PreferenceManager.fromContext(activity).setLockTimestamp(System.currentTimeMillis())`

Comment (ezaquarii) [reply src/main/java/com/owncloud/android/authentication/PassCodeManager.java]: Fixed

File: src/main/java/com/owncloud/android/authentication/PassCodeManager.java (hunk 1 above)
Comment (ezaquarii) [reply src/main/java/com/owncloud/android/authentication/PassCodeManager.java]: Fixed

**Pull request merged**